### PR TITLE
CA-2003 fix audiorecord

### PR DESCRIPTION
**Pull request by Tooseriuz**
### Changes

- Comment audio read (that should be the one that make audioRecorder break)
- Add Wakelock (that should make all services are up all the time)

### CODE REVIEW

File: guardian/src/main/java/org/rfcx/guardian/guardian/RfcxGuardian.java [hunk 1]
```diff
@@ -154,6 +162,7 @@ public void appPause() {
     public void startAllServices() {
         if (isRequirementPassed()) {
             if(!getRecordingState()){
+                Log.d(logTag,"start");
```
Comment (antonyharfield) [inline guardian/src/main/java/org/rfcx/guardian/guardian/RfcxGuardian.java]: Is this neccessary?

File: guardian/src/main/java/org/rfcx/guardian/guardian/receiver/BootReceiver.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ public void onReceive(Context context, Intent intent) {
 		
 		// initializing rfcx application
 		RfcxGuardian app = (RfcxGuardian) context.getApplicationContext();
-		app.startAllServices();
+//		app.startAllServices();
```
Comment (antonyharfield) [inline guardian/src/main/java/org/rfcx/guardian/guardian/receiver/BootReceiver.java]: Please delete, not comment.

File: guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java [hunk 3]
```diff
@@ -304,7 +304,7 @@ public void startRecorder() {
 
 			captureFilePayloadSizeInBytes = 0;
 			audioRecorder.startRecording();
-			audioRecorder.read(uncompressedOutputBuffer, 0, uncompressedOutputBuffer.length);
+//			audioRecorder.read(uncompressedOutputBuffer, 0, uncompressedOutputBuffer.length);
```
Comment (antonyharfield) [inline guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java]: Also delete. :)

Comment (antonyharfield) [reply guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java]: Was this causing the bug? Or was it related to wake lock?

**Review by antonyharfield — CHANGES_REQUESTED** [review]
Tidy up the code, remove comments.

Do you know why the `audioRecorder.read()` was in there?

File: guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java (hunk 3 above)
Comment (Tooseriuz) [reply guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java]: > Was this causing the bug? Or was it related to wake lock?

This cause the bug, it was from Topher's first code.
For the wake lock I added later and test it but still error.
After that I comment this calling and it works.
So Im not sure if it just comment this line that solve the bug or both with wake lock and comment.

**Comment by Tooseriuz** [comment]
I will push it with the removing of wake lock.

**Review by antonyharfield — APPROVED** [review]

**Pull request merged**